src/usecases/retrieveWardVisitTotals.test.js

```diff
@@ -37,8 +65,51 @@ describe("retrieveWardVisitTotals", () => {
       expect(res).toEqual({
         total: 30,
         byWard: [
-          { hospitalName: "Hospital One", name: "Ward One", visits: 10 },
-          { hospitalName: "Hospital One", name: "Ward Two", visits: 20 },
+          {
+            hospitalName: "Hospital One",
+            name: "Ward One",
+            visits: 10,
+            trustId: 0,
+          },
+          {
+            hospitalName: "Hospital One",
+            name: "Ward Two",
+            visits: 20,
+            trustId: 1,
+          },
+        ],
+      });
+    });
+
+    it("Returns the totals per ward and the overall total for the specified trust", async () => {
+      const anySpy = jest.fn(async () => [
+        {
+          hospital_name: "Hospital One",
+          name: "Ward Two",
+          total_visits: "20",
+          trust_id: 1,
+        },
+      ]);
+
+      const container = {
+        getDb: async () => ({
+          any: anySpy,
+          trustId: 1,
+        }),
+      };
+
+      let res = await retrieveWardVisitTotals(container)();
```

Comment:
I think we need to pass in the `trustId` here / should explicitly do it in the test.

```suggestion
      const trustId = 1;
      let res = await retrieveWardVisitTotals(container)(trustId);
```